FordLabs/PeopleMover
# Landing Page Accessibility

Co-authored-by: Ashley Clifton <aclift19@ford.com>
Co-authored-by: chris dorman <cdorman1@ford.com>

## Issue
Resolves #78 Accessibility: Landing Page

## What was done
- [x] Added alt text to images (the main image now says `Preview of People Mover Space after logging in`)
- [x] Updated the People Mover logo to be grouped so the screen reader reads "People Mover Logo, Group". The PeopleMover logo is no longer an h1.
- [x] Updated the Powered by FordLabs logo to be grouped so the screen reader reads "Powered By Ford Labs, Group" 
- [x] Updated the Landing page so that the logo is *not* an h1, but `It’s about the people. Your people.` is an h1, and `And helping them be extraordinary.` is an h2, and the paragraph is wrapped in a p tag.
- [x] Updated the getting started button so that it's only an a tag (not a button in an a tag) and so that the focus highlight actually borders the element.

## How to test
1. Visit landing page with screen reader and ensure you can enter the main content and navigate around and that the screen reader gives you helpful verbiage about what you're viewing.

### FOR DEVS
Feature URL: See feature branch

### FOR APPROVAL
Approval URL: See Approval branch
